ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/event_handlers.schema.yml

```diff
@@ -42,6 +42,55 @@ keys:
           - on-boot
           - on-logging
           - on-startup-config
+          - on-maintenance
+        trigger_on_maintenance:
+          description: Settings required for trigger 'on-maintenance'.
+          type: dict
+          keys:
+            operation:
+              type: str
+              required: true
+              valid_values:
+              - "enter"
+              - "exit"
+            bgp:
+              description: Trigger condition occurs on maintenance operation of specified BGP peer.
+              type: dict
+              keys:
+                neighbor_address_ipv4:
+                  type: str
+                neighbor_address_ipv6:
+                  type: str
+                peer_group_name:
+                  type: str
+            stage:
+              type: str
+              required: true
+              description: Stages of maintenance operation. "vrf" stage can be defined for "bgp" only.
+              valid_values:
+              - after
+              - before
+              - all
+              - begin
+              - end
+              - vrf
```

Comment:
please note that after vrf also we can trigger the all other action

Reply:
fixed